### PR TITLE
[CDAP-8427] - Consolidate color scheme for Tracker/Hydrator/Wrangler

### PR DESCRIPTION
**Pull request by sushil-car**
https://issues.cask.co/browse/CDAP-8427

### CODE REVIEW

**Comment by ajainarayanan** [comment]
Added on-hold as verifying with @bdmogal and Lea on color/theme change throughout metadata section. There are other places where we still use tracker green that is not documented in the JIRA.

**Comment by sushiladokar** [comment]
new color scheme

**Comment by sushiladokar** [comment]
wrangle modal

**Comment by sushil-car** [comment]
audit graph hover
jump button hover
new color scheme
user system tags

**Comment by sushil-car** [comment]
deletepipelineconfirm

**Comment by sushil-car** [comment]
tracker search page

**Review by bdmogal — APPROVED** [review]
lgtm, reviewed the colors with @ajainarayanan

File: cdap-ui/app/tracker/templates/metadata.html [hunk 1]
```diff
@@ -26,7 +26,7 @@
   <div class="entity-detail-tags-column">
     <div class="entity-container">
       <div class="user-tags">
-        <h4>User Tags</h4>
+        <h4>Business Tags</h4>
```
Comment (elfenheart) [inline cdap-ui/app/tracker/templates/metadata.html:29]: Line 150 needs to be change to `Business` as well

**Comment by elfenheart** [comment]
👍  LGTM

**Pull request merged**